docs/guides/Redux-Integration.md

```diff
@@ -55,7 +55,7 @@ class Root extends React.Component {
 
 Once you do this, your navigation state is stored within your redux store, at which point you can fire navigation actions using your redux dispatch function.
 
-Keep in mind that when a navigator is given a `navigation` prop, it relinquishes control of its internal state. That means you are now responsible for persisting its state, handling any deep linking, integrating the back button, etc.
+Keep in mind that when a navigator is given a `navigation` prop, it relinquishes control of its internal state. That means you are now responsible for persisting its state, handling any deep linking, integrating the back button (see below extra), etc.
```

Comment:
Nit: add an [anchor link in markdown](https://gist.github.com/asabaylus/3071099) to the wording in the parens.

I would also change "below extra" to the name of the section header. E.g.: "see Handling Hardware Back Button in Android)".